src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String block; 
+	public Block (String block){
+		this.block=block;
+	}
+	public String getBlock (){
+		return block;
```

Comment:
noticed that the `return` statement of other classes' getter methods generally uses the `this` keyword to refer to a class level variable. Consider deciding on a style and stick to it.
i.e. `return this.street;`